docs/monitoring.md

```diff
@@ -312,15 +312,39 @@ can be identified by their `[attempt-id]`. In the API listed below, when running
     <td>Details for the storage status of a given RDD.</td>
   </tr>
   <tr>
-    <td><code>/applications/[base-app-id]/logs</code></td>
```

Comment:
Don't change this. This is correct and matches the definition of `base-app-id` in the documentation.